sympy/sets/sets.py

```diff
@@ -216,7 +216,17 @@ def _complement(self, other):
             return S.EmptySet
 
         elif isinstance(other, FiniteSet):
-            return FiniteSet(*[el for el in other if self.contains(el) != True])
+            from sympy.utilities.iterable import sift
```

Comment:
can these go at the top of the file?

Reply:
I didn't get you... What did you mean ?

Reply:
Can you tell me why I am having a merge conflict here?